modules/ROOT/pages/traversal-framework/traversal_framework_java_api.adoc

```diff
@@ -9,22 +9,24 @@
 = Traversal Framework Java API
 
 [[traversal-java-api-traversaldescription]]
-== link:{org-neo4j-graphdb-traversal-description}[TraversalDescription^]
+== `TraversalDescription`
 
-The `TraversalDescription` is the main interface used for defining and initializing traversals.
-It is not meant to be implemented by users of the Traversal Framework, but rather is provided as a way for the user to describe traversals.
-`TraversalDescription` instances are immutable and its methods return a new `TraversalDescription` that is modified compared to the object the method was invoked on with the arguments of the method.
+`TraversalDescription` is the main interface used for defining and initializing traversals.
+It is not meant to be implemented by users of the Traversal Framework, but it works as a means for the user to describe traversals.
+
+`TraversalDescription` instances are immutable.
+Their methods return a new `TraversalDescription` that is modified according to the object that the method was invoked on, with the arguments of it.
 The method `traverse()` returns the result of the traversal defined in the `TraversalDescription`.
 
 [[traversal-java-api-traverser]]
-== Traverser
+== `Traverser`
```

Comment:
The titles on this page which are in codeblocks look a little strange - perhaps making the title longer may help?

```suggestion
== The `Traverser` object
```